.vscode/launch.json

```diff
@@ -12,7 +12,8 @@
             "env": {
                 "token": "",
                 "prefix": "",
-                "ownerID": ""
+                "ownerID": "",
+                "github": { "token": "", "username": "", "repoName": "" }
```

Comment:
For the token here you can follow [these instructions](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line)